index.html

```diff
@@ -17,7 +17,7 @@ <h2>Select your media in Emby and play it here</h2>
         <div class="timeOfDay"><span class="timePrefix"></span> <span class="timeSuffix"></span></div>
     </div>
     <video id="video-player" crossorigin="anonymous" preload="auto"></video>
-    <img id="photo"></img>
+    <img id="photo">
```

Comment:
Should use a self-closing tag (`<img id="photo"/>`).

Reply:
Hmm, wonder why VSCode didn't complain about that.